ansible/roles/jonatasbaldin.zabbix3agent/.gitignore

```diff
@@ -0,0 +1 @@
+.vagrant
```

Comment:
Why are we adding this to our repo? Can't we use [Ansible Galaxy](https://galaxy.ansible.com/intro)?